wingtips-core/src/main/java/com/nike/wingtips/Span.java

```diff
@@ -316,7 +308,7 @@ public SpanPurpose getSpanPurpose() {
      * <p/>
      * NOTE: This is intentionally package scoped to make sure completions and logging/span output logic happens centrally through {@link Tracer}.
      */
-    void complete() {
+    public void complete() {
```

Comment:
make a comment why public as it violates the comment above

Reply:
I checked out the code, and from what I can see this was just done because parsing logic is no longer in the same package, so it doesn't see `Span.complete()` anymore. This should not be made public - there are ways to do what you need in the tests without calling `span.complete()` directly. Easiest is probably to just use `Tracer` to generate the spans and complete them.

Reply:
I reverted to package visibility.  I created a class `TestSpanCompleter` that is in the same package as `Span` and exposes a static method to close a test instance of a `Span`.   I call that method from the tests that need a closed `Span`.  It seemed a little cleaner to keep `Tracer` and `Span` testing separate. I'm open to feedback on this approach.